cli/pkg/kctrl/cmd/package/release/release.go

```diff
@@ -32,6 +32,7 @@ type ReleaseOptions struct {
 	repoOutputLocation    string
 	debug                 bool
 	generateOpenAPISchema bool
+	tag                   string
```

Comment:
We are using `bundleTag` in one place and `tag` in another. Should we keep it consistent?

Reply:
We are using it as tag only during releaseOptions, When we are inside imgpkg, we are using bundleTag. Tomorrow, if we start supporting other mechanisms, we may reuse tag.

Reply:
I think we are using `bundleTag` in app release as well.
I think we could go the other way around and change the variable name if we do reuse the tag flag

Reply:
Changed `bundleTag` to `tag` in app release.